sdk/storage/azure-storage-file-datalake/samples/datalake_samples_access_control_recursive_async.py

```diff
@@ -58,9 +60,19 @@ async def progress_callback(acl_changes):
         failed_entries.append(acl_changes.batch_failures)
 
     # illustrate the operation by using a small batch_size
-    acl_change_result = await directory_client.set_access_control_recursive(acl=acl,
-                                                                            progress_callback=progress_callback,
-                                                                            batch_size=5)
+    try:
+        acl_change_result = await directory_client.set_access_control_recursive(acl=acl,
+                                                                                progress_callback=progress_callback,
+                                                                                batch_size=5)
+    except AzureError as error:
+        # if the error has continuation_token, you can restart the operation using that continuation_token
+        if error.continuation_token:
+            acl_change_result = \
+                await directory_client.set_access_control_recursive(acl=acl,
+                                                                    continuation_token=error.continuation_token,
+                                                                    progress_callback=progress_callback,
```

Comment:
I thought this parameter had been renamed to `progress_hook`?

Reply:
Thanks for catching this, the parameter in this sample was missed during renaming.